eth/catalyst/api.go

```diff
@@ -301,8 +305,8 @@ func (api *ConsensusAPI) assembleBlock(parentHash common.Hash, params *PayloadAt
 		return nil, fmt.Errorf("cannot assemble block with unknown parent %s", parentHash)
 	}
 
-	if params.Timestamp < parent.Time() {
-		return nil, fmt.Errorf("child timestamp lower than parent's: %d < %d", params.Timestamp, parent.Time())
+	if params.Timestamp <= parent.Time() {
+		return nil, fmt.Errorf("child timestamp lower or equal to parent's: %d < %d", params.Timestamp, parent.Time())
```

Comment:
Writing out literally "lower or equal to" and then also writing the same things as `<` is redundant. (Also `<` does not contain the equality). It's a lot simpler to write as `"invalid timestamp: child's %d <= parent's %d"`